src/vendor-dashboard/reports/layout/style.scss

```
@@ -4,277 +4,277 @@
+        }
     }
-  }
 
-  .woocommerce-usage-modal__actions {
-    display: flex;
-    justify-content: flex-end;
-    // margin-top: $gap;
+    .woocommerce-usage-modal__actions {
+        display: flex;
+        justify-content: flex-end;
+        // margin-top: $gap;
```

Stylelint (17.2.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)